draft/draft-ietf-ippm-twamp-yang.xml

```diff
@@ -1524,15 +1524,18 @@ module ietf-twamp {
           }
 
           leaf reflector-udp-port {
-            type dynamic-port-number;
+            type uint32 {
+              range "862|49152..65535";

```

Comment:
The syntax looks fine. But when you say 'it', I guess you mean pyang crashed. Have you tried using yanglint to generate the tree? Does it also crash?

Reply:
Thanks for jumping to my rescue, Mahesh! 
I use eclipse for all the YANG stuff, with plug-ins that Kevin D'Sousa wrote. 
My Eclipse install doesn't seem to be fully functional today. it doesn't even save
the YANG file when asked.

Reply:
BTW, you need to make this change in the model in the src/yang directory. The make run copies the model into the draft's .xml file. The way your changes have been made, they will be overwritten when a make is run.